students/brandon_nguyen/session03/slicing_lab.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+import unittest
+
+#slicing lab in lession 3
+#Student Brandon Nguyen
+
+"""
+Exchange first and last sequence.
+"""
+def exchange_first_last(seq):
+    return seq[-1:]+seq[1:len(seq)-1]+seq[0:1]
```

Comment:
Minor: No need to use `len(seq)-1`. Just use `-1`, and Python runtime will automatically convert it to `len(seq)-1`.

Likewise, no need to specify `0` as the starting index. Python runtime will set the starting index to 0 if none is specified.

Reply:
Thank you!